test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh

```diff
@@ -5,7 +5,8 @@
 set -e
 
 readonly DEFAULT_VALIDITY_DAYS=${DEFAULT_VALIDITY_DAYS:-730}
-readonly HERE=$(cd "$(dirname "$0")" && pwd)
+readonly HERE
+HERE=$(cd "$(dirname "$0")" && pwd)
```

Comment:
shouldn't it be the other way around? assign first, then mark it readonly?

Reply:
i wasnt sure about that - but shellcheck didnt complain at this - i can try the other way...

Reply:
works setting first - my intuition was that readonly in bash is set once - but ill swap these around